### ISSUE TITLE
Duplicated scripts

### ISSUE THREAD

**Issue by wojtek717** [withfig/fig#2263]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.1.0 (22C65)

### Expected behaviour

No duplicated scripts in the terminal.

### Actual behaviour

Duplicated scripts in the terminal
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/22446225/215046912-894740a4-39f2-499d-9ba0-9cafa4a532f6.png">

### Steps to reproduce

Run `fig run`
See duplicated scripts

### Environment

```yaml
fig-details:
  - 2.13.1
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.1.0 (22C65)
environment:
  - shell: /bin/zsh
  - terminal: terminal
  - cwd: /Users/wojciechkonury
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - FIGTERM_SESSION_ID: 0560d0da-2b8c-4f83-9d3a-6d8f02460826
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.13.1
    - PATH: /Users/wojciechkonury/.rvm/gems/ruby-3.0.0/bin:/Users/wojciechkonury/.rvm/gems/ruby-3.0.0@global/bin:/Users/wojciechkonury/.rvm/rubies/ruby-3.0.0/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/wojciechkonury/.fig/bin:/Users/wojciechkonury/.local/bin:/Users/wojciechkonury/.rvm/bin:/Users/wojciechkonury/flutter/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.apple.Terminal
    - FIG_PID: 83285
    - FIG_SET_PARENT: 0560d0da-2b8c-4f83-9d3a-6d8f02460826
```

**Comment by mschrage** [comment]
Thanks for reporting @wojtek717! We have a fix that will go live in the next day or two.

**Comment by grant0417** [comment]
This should be resolved now in 2.14
